style.css

```diff
@@ -0,0 +1,800 @@
+
+/* Mobile View design */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
+/* Top Navigation Bar and Second Navigation */
+.first-navigation {
+  display: none;
+}
+
+.sec-ul li {
+  padding: 19px;
+  display: flex;
+}
+
+.sec-nav span {
+  display: block;
+}
+
+#close-btn {
+  display: block;
```

Comment:
- [ ] You should always use the class attribute for multiple elements (you shouldn't  ID selector for multiple elements). You use same id `"close-btn"` in html. Please fix this.